package.json

```diff
@@ -6,7 +6,7 @@
     "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
     "@rails/webpacker": "5.4.0",
-    "bootstrap": "^5.1.0",
+    "bootstrap": "4.0.0-alpha.6",
```

Comment:
why the version change?

Reply:
By mistake I made the change before, initially, I work with bootstrap 4.